src/logging.cc

```diff
@@ -2132,7 +2134,14 @@ string LogSink::ToString(LogSeverity severity, const char* file, int line,
          << ' '
          << file << ':' << line << "] ";
 
-  stream << string(message, message_len);
+#ifdef _MSC_VER
+#pragma push_macro("write")
+#undef write
+#endif  // _MSC_VER
```

Comment:
The error was
> '_write': is not a member of 'std::ostringstream
https://buildkite.com/bazel/google-logging/builds/2869#01829651-f13d-4665-9c42-dfd29c8ac36a

The source of the problem is
```cpp
src/windows/port.h:87:#define write   _write
```

Reply:
Thanks. Instead of undefing the macro, you can prevent its expansion by protecting the `write` calls as follows:
```cpp
(stream.write)(message, static_cast<std::streamsize>(message_len));
```
Please also add a comment for the reason why macro expansion is prevented.

Reply:
OK, thank you.  That would require redefining `write` macro with three parameters.

Reply:
Good point. I missed that.